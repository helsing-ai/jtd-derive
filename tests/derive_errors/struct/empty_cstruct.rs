#[derive(jtd_derive::JsonTypedef)]
struct EmptyCstruct {}

fn main() {}
