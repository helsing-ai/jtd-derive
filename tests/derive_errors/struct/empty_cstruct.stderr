error: jtd-derive does not support cstruct-like structs
 --> tests/derive_errors/struct/empty_cstruct.rs:2:8
  |
2 | struct EmptyCstruct {}
  |        ^^^^^^^^^^^^
